pkg/frontend/metrics.go

```diff
@@ -0,0 +1,35 @@
+package frontend
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"time"
+
+	"github.com/Azure/ARO-RP/pkg/util/recover"
+)
+
+//emitMetrics sends a heartbeat metric every 60 seconds
+func (f *frontend) emitMetrics(stop <-chan struct{}) {
```

Comment:
We also need the same functionality for the monitoring - can you make this more generic and handle both cases in this PR?

Reply:
Moved the code to /pkg/util/heartbeat and added it in monitor.